central/administration/events/service/service_impl.go

```diff
@@ -114,17 +115,17 @@ func getQueryBuilderFromFilter(filter *v1.AdministrationEventsFilter) *search.Qu
 			// events to listing them is relatively short.
 			protoconv.ConvertTimestampToTimeOrDefault(filter.GetUntil(), time.Now().Add(time.Second)),
 		)
-	if domain := filter.GetDomain(); domain != "" {
-		queryBuilder = queryBuilder.AddExactMatches(search.EventDomain, domain)
+	if domains := filter.GetDomain(); len(domains) != 0 {
+		queryBuilder = queryBuilder.AddExactMatches(search.EventDomain, domains...)
```

Comment:
Do we need to validate for duplicated values here?

Reply:
I can de-dupe the list, note that since this is a disjunction query it doesn't necessarily matter if there are duplicates.